xarray/tests/test_formatting.py

```diff
@@ -472,6 +474,25 @@ def test_large_array_repr_length():
     assert len(result) < 50
 
 
+@requires_netCDF4
+def test_repr_file_collapsed(tmp_path):
+    arr = xr.DataArray(np.random.randn(100), dims="test")
```

Comment:
we don't actually need the values so it's cheaper to just use range (it's also a tiny bit faster):
```suggestion
    arr = xr.DataArray(np.arange(100), dims="test")
```